crates/ruff_python_formatter/src/expression/parentheses.rs

```diff
@@ -173,12 +187,14 @@ impl<'ast> Format<PyFormatContext<'ast>> for FormatParenthesized<'_, 'ast> {
             // This ensures that expanding this parenthesized expression does not expand the optional parentheses group.
             write!(
                 f,
-                [fits_expanded(&inner)
+                [fits_expanded(&group(&inner))
```

Comment:
I would expect that content with `ungroup = true` never gets grouped. Meaning, we should only use `fits_expanded(&group(&inner))` is false.